mods/default/furnace.lua

```diff
@@ -224,6 +223,11 @@ local function furnace_node_timer(pos, elapsed)
 		elapsed = elapsed - el
 	end
 
+	if items_smelt ~= 0 then
+		-- Play cooling sound
+		minetest.sound_play("default_cool_lava",
+			{ pos = pos, max_hear_distance = 16, gain = 0.07 * math.min(items_smelt, 7) }, true)
```

Comment:
Makes sense to not let it get arbitrarily loud; how did you choose this? I assume you roughly went for ~0.5 max gain?

Reply:
At first, I went for 0.05, realizing that it works but was too quiet, so I increased that to 0.07.